hack/make-rules/clean.sh

```diff
@@ -24,15 +24,14 @@ source "${KUBE_ROOT}/hack/lib/util.sh"
 CLEAN_PATTERNS=(
   "_tmp"
   "doc_tmp"
-  "((?!staging\/src\/k8s\.io\/apiextensions-apiserver\/pkg\/generated\/openapi).)*/zz_generated.openapi.go"
```

Comment:
what happened with this pattern?

Reply:
It matches nothing - all openapi files are properly checked in now (for a while, actually)